src/main/java/utils/reflection/ReflectionUtilities.java

```diff
@@ -375,6 +372,24 @@ public static Object getField(String fieldName, Object inputObject) {
         }
     }
 
+    /**
+     * This method set the value to a field of object with a given field name.
+     *
+     * @param object     The object in which field the value is set.
+     * @param fieldName  The name of the field to set the value.
+     * @param fieldValue The value to set the field of object.
+     * @throws RuntimeException If the field cannot be accessed or does not exist.
+     */
+    public static void setObjectField(Object object, String fieldName, Object fieldValue) {
```

Comment:
lets rename this wonderful utility setField()

Reply:
done